AsyncGenerator.yml

```diff
@@ -28,259 +28,6 @@
   registerPlugin:
   - type: AsyncGenerator.Core.Plugins.EmptyRegionRemover
     assemblyName: AsyncGenerator.Core
-- filePath: CoreMemoryCache\NHibernate.Caches.CoreMemoryCache\NHibernate.Caches.CoreMemoryCache.csproj
```

Comment:
Most caches do no more need an async generation, as their cache implementation do not support async. The inherited default async handling is then suitable for them.